src/test/utils/gameManager.ts

```diff
@@ -458,6 +459,24 @@ export default class GameManager {
     });
   }
 
+  doThrowPokeball(ballIndex: number) {
```

Comment:
```suggestion
  public doThrowPokeball(ballIndex: number): void {
```
Should have some tsdocs.

Reply:
Actually, writing the tsdocs made me realise there was no point to having two separate functions for the ball throw, so I merged them.